lib/util.js

```diff
@@ -119,12 +119,14 @@ const util = {
       parseMatch();
     }
 
-    return ({
+    const result = {
       command: command,
       match: match,
       matchArgs: matchArgs,
       pipes: pipes
-    });
```

Comment:
Yes, you have it write. The entire command execution code including pipe implementation should probably be re-written if we are going to use bash-parser. It needs a re-think from the ground up.